rita_common/src/usage_tracker/mod.rs

```diff
@@ -130,6 +170,7 @@ impl UsageTracker {
         loop {
             match file.write_all(&compressed_bytes) {
                 Ok(save) => {
+                    info!("Saved to disk for usage tracker");
```

Comment:
add the number of bytes compressed_bytes.len() to this message. as well as all saving log messages